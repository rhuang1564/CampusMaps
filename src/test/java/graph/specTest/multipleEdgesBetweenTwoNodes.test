# Tests adding a multiple edges between same node

# Create a graph
CreateGraph A

# Add a node
AddNode A node1
AddNode A node2

# Add edges
AddEdge A node1 node2 edge1
AddEdge A node1 node2 edge2
AddEdge A node1 node2 edge3
AddEdge A node1 node2 edge4
AddEdge A node2 node1 edge1
AddEdge A node2 node1 edge2
AddEdge A node2 node1 edge3
AddEdge A node2 node1 edge4
AddEdge A node2 node1 edge5

# Print the nodes in the graph and checks its edges
ListNodes A
ListChildren A node1
ListChildren A node2