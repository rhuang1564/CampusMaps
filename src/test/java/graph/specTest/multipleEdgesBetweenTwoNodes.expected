# Tests adding a multiple edges between same node

# Create a graph
created graph A

# Add a node
added node node1 to A
added node node2 to A

# Add edges
added edge edge1 from node1 to node2 in A
added edge edge2 from node1 to node2 in A
added edge edge3 from node1 to node2 in A
added edge edge4 from node1 to node2 in A
added edge edge1 from node2 to node1 in A
added edge edge2 from node2 to node1 in A
added edge edge3 from node2 to node1 in A
added edge edge4 from node2 to node1 in A
added edge edge5 from node2 to node1 in A

# Print the nodes in the graph and checks its edges
A contains: node1 node2
the children of node1 in A are: node2(edge1) node2(edge2) node2(edge3) node2(edge4)
the children of node2 in A are: node1(edge1) node1(edge2) node1(edge3) node1(edge4) node1(edge5)